common/djangoapps/student/tests/test_models.py

```diff
@@ -383,3 +384,20 @@ def test_retirement(self):
         self.assertFalse(ManualEnrollmentAudit.objects.filter(enrollment=enrollment).exclude(
             reason=""
         ))
+
+
+class TestAccountRecovery(TestCase):
+    """ Tests for the AccountRecovery Model """
```

Comment:
Nit: our established docstring coding style puts the triple-quotes on their own lines. https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
thanks! I didn't realize